components/service/glean/src/main/java/mozilla/components/service/glean/config/Configuration.kt

```diff
@@ -46,13 +46,14 @@ data class Configuration internal constructor(
     // constructor and only initialized with a proper default when calling the primary
     // constructor from the secondary, public one, below.
     constructor(
+        serverEndpoint: String = DEFAULT_TELEMETRY_ENDPOINT,
```

Comment:
I knew that this would eventually happen, but it's still a little scary to open up the server endpoint to the public constructor.

Reply:
Yup, but think of this: the only way to change this would be to recompile the app. At that point, if recompilation is required, then anything can happen.

As long as this is not a runtime setting, we should be fine.